src/repror/internals/commands.py

```diff
@@ -42,24 +42,24 @@ def run_streaming_command(
     # We can only capture of stdout or stderr, not both
     # Otherwise it will block
     # See: https://stackoverflow.com/questions/18421757/live-output-from-subprocess-command
-
     if stream_type.is_stderr:
         kwargs = {"stderr": subprocess.PIPE}
     else:
         kwargs = {"stdout": subprocess.PIPE}
 
-    with subprocess.Popen(command, cwd=cwd, env=env, **kwargs) as process:
-        # Print stderr as it comes in
-        for line in io.TextIOWrapper(
-            process.stderr, encoding="utf-8"
-        ):  # or another encoding
-            print(line, end="")
+    with subprocess.Popen(args=command, cwd=cwd, env=env, **kwargs) as process:
+        main_stream = process.stderr if stream_type.is_stderr else process.stdout
```

Comment:
let write it :
`main_stream, other_stream = process.stderr, process.stdout if ... else ...`